test/inductor/test_aot_inductor.py

```diff
@@ -1454,5 +1453,5 @@ class AOTInductorTestNonABICompatibleCuda(TestCase):
     from torch._dynamo.test_case import run_tests
 
     # cpp_extension N/A in fbcode
-    if HAS_CUDA and not TEST_WITH_ROCM:
+    if not TEST_WITH_ROCM:
```

Comment:
You need to make sure all the cuda tests are decorated with `@requires_cuda` if you want to remove this.